### DISCUSSION TITLE
Highlight Spaces in Path

### DISCUSSION THREAD

**Discussion by JPLuehrig** [cmderdev/cmder#2573] [General]
Hey there,

I have the problem that my cmder does not show the coloring and the additional infos... it looks like the cmd. 
I Started the cmder from the downloads folder, which has unfortunately some spaces inside... can we highlight this more, because afterwards all functionality is given, when I moved that to an other folder? In the documentation is that not highlighted.

Best regards JP Lührig

**Comment by daxgames** [comment]
I don't understand.  Are you saying that Cmder does not work in a path with spaces in it?  I routinely test it from a path with spaces and have never had an issue.

**Reply by JPLuehrig** [reply]
I am sorry, I unpacked the Cmder in a folder where a parent folder has a space inside. 

C:\Tools\Spaced Name\Cmder\...

Then the cool and fancy visualizations do not work and looks like a normal cmd.

**Comment by daxgames** [comment]
Below is a screenshot of my Cmder running 1.3.18:

![image](https://user-images.githubusercontent.com/7318053/126855157-4ef32e5e-b0f4-45e8-974f-7c32d8ac4504.png)

Seems to work fine with a space in the app parent folder path.

**Reply by JPLuehrig** [reply]
Ok, I'am sorry, I do not investigate... now I did, and it has something to do with the special sign:

![image](https://user-images.githubusercontent.com/86967122/126946388-64bdfdbe-9603-46e7-b8e2-79c330b20af8.png)

My name has a special case and something allowed in the folder like ÄÜÖ does break it

Therefore the downloads folder was in the Directory C:\Users\JP Lührig\Downloads

**Comment by mrx23dot** [comment]
Try running cmder.exe it should give you coloring.
If that works you can make it work from context menu:
https://github.com/cmderdev/cmder/issues/2597
